collect_app/src/main/java/org/odk/collect/android/application/initialization/ApplicationInitializer.kt

```diff
@@ -80,6 +83,9 @@ class ApplicationInitializer(
             CollectSetGeopointActionHandler.ELEMENT_NAME,
             CollectSetGeopointActionHandler()
         )
+
+        // Configure default parser factory
+        XFormUtils.setXFormParserFactory(EntityXFormParserFactory(XFormParserFactory()))
```

Comment:
Do we need this in Collect? Can't it be set in javarosa?

Reply:
We decided in https://github.com/getodk/javarosa/pull/691 to try and make entities a "plug-in" feature for JavaRosa rather than add more logic to the core parsing/processing to deal with them. We're planning to do the same thing for the geo features (like parsing a geopoint out of a form). That's discussed more in the linked PR.

I do agree this setup seems a little clumsy though. I reckon we probably want to eventually get rid of the "default" `XFormParserFactory` and introduce something like a builder that allows you to add features like entities.

Reply:
Ok I see.